selinux/label_darwin.go

```diff
@@ -0,0 +1,30 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
we need the per-plaform impls because we will use selinux in the cmd/snap code, right?

Reply:
yes